CompuCell3D/core/Demos/CallableCC3D/cc3d_render/cc3d_render.py

```diff
@@ -0,0 +1,35 @@
+from os.path import dirname, join
+from cc3d.CompuCellSetup.CC3DCaller import CC3DCaller
+from cc3d.CompuCellSetup.CC3DRenderer import CC3DRenderer, standard_lds_file, standard_screenshot_file
+
+sim_name = 'cellsort_2D.cc3d'
+output_frequency = 40
+render_dir = 'Figs'
+
+
+def main():
+    cc3d_sim_fname = join(dirname(__file__), sim_name)
```

Comment:
Since this is a demo I'd put a lot of comments explaining what (almost) every line of code is doing

Reply:
Done. See https://github.com/CompuCell3D/CompuCell3D/pull/355/commits/8617ebd1c8df5e1fb876d899aeb98d897eca9ee5